windows/src/desktop/help/about/requirements.md

```diff
@@ -4,20 +4,18 @@ title: System Requirements
 
 ## Supported Windows Operating Systems
```

Comment:
Should we remove this duplicated content and refer to common/os.md instead?

Reply:
Sure. I'll apply all the suggestions first. Then link to os.md